lib/mrsk/cli/base.rb

```diff
@@ -73,7 +73,9 @@ def print_runtime
       end
 
       def audit_broadcast(line)
-        run_locally { execute *MRSK.auditor.broadcast(line), verbosity: :debug }
+        if broadcast = MRSK.auditor.broadcast(line)
+          system(MRSK.auditor.broadcast_environment(line), broadcast)
```

Comment:
I don't think there's a way to set the environment when using `run_locally`, so I switched to `system` for this one instead.

Reply:
@kevinmcconnell Did you want to switch to this #with approach?